server/package.json

```diff
@@ -9,7 +9,7 @@
     "build:mock": "rimraf dist && ncc build src/mockServer.ts -o dist",
     "dev:server": "nodemon",
     "dev:server-debug": "node -r ts-node/register --inspect=5858 src/index.ts",
-    "test": "jest"
+    "test": "jest --detectOpenHandles"
```

Comment:
Trenger vi denne permanent? Fra Jest-dok: "This option has a significant performance penalty and should only be used for debugging."

Reply:
Nei, den var til debugging bare. Fjerner